tools/sense_studio/sense_studio.py

```diff
@@ -337,6 +338,40 @@ def show_video_list(split, label, path):
     return render_template('video_list.html', folders=folder_id, split=split, label=label, path=path)
 
 
+@app.route('/record_video/<split>/<label>/<path:path>')
+def record_video(split, label, path):
+    """
+    Show the list of videos for the given split, class label and project.
+    If the necessary files for annotation haven't been prepared yet, this is done now.
+    """
+    path = f'/{urllib.parse.unquote(path)}'  # Make path absolute
+    split = urllib.parse.unquote((split))
+    label = urllib.parse.unquote(label)
+    return render_template('record_video.html', split=split, label=label, path=path)
+
+
+@app.route('/video_saving/<split>/<label>/<path:path>', methods=['POST'])
+def save_video(path, label, split):
+    path = f'/{urllib.parse.unquote(path)}'  # Make path absolute
```

Comment:
`label` and `split` should also be unquoted

Reply:
I am doing it, but what does it change ? It was working before without it

Reply:
This only gets problematic, when your class names contain spaces